### DISCUSSION TITLE
How do you move the layers panel?

### DISCUSSION THREAD

**Discussion by ohyearsonist** [LibreSprite/LibreSprite#361] [Q&A]
In Aseprite, you can click the configuration button in the layers panel and move it to the right, which makes a lot more sense to me. However, I didn't find any positioning options in LibreSprite. Is that just a feature not yet implemented or am I missing something?

**Comment by Ale3ex** [comment]
Similar question. I have a widescreen monitor and I really don't like how the layers panel takes up so much space at the bottom of the screen when it would look great on the side. I suspect that the ability to move the layers panel is simply not present in LibreSprite.

**Comment by fccfc5d245c** [comment]
Using Tab to show and then hide the panel does the job for me.

**Comment by felipemanga** [comment]
On 1.1 you can click on "View" then "Alternate Timeline"

**Comment by LadyZora** [comment]
> In Aseprite, you can click the configuration button in the layers panel and move it to the right, which makes a lot more sense to me. However, I didn't find any positioning options in LibreSprite. Is that just a feature not yet implemented or am I missing something?

I did some digging and messed with the settings. You can move the Layers and Tools by clicking on View -> Alternate Timeline and Alternate Toolbar! They will stay the same as you have set them even when you open a new Sprite Window!
